<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tool_container">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>461</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>651</width>
      <height>461</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ascii_hex_binary">
     <attribute name="title">
      <string>CodeFormatter</string>
     </attribute>
     <widget class="QGroupBox" name="input_groupBox">
      <property name="geometry">
       <rect>
        <x>448</x>
        <y>10</y>
        <width>91</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>In_Format</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>87</width>
         <height>136</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rb_input_hex">
          <property name="text">
           <string>Hex</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_input_decimal">
          <property name="text">
           <string>Decimal</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_input_ascii">
          <property name="text">
           <string>Ascii</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_input_oct">
          <property name="text">
           <string>Oct</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_input_binary">
          <property name="text">
           <string>Binary</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="Input_Label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Input</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="InputEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>47</y>
        <width>421</width>
        <height>141</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
     <widget class="QLabel" name="output_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="output_groupBox">
      <property name="geometry">
       <rect>
        <x>546</x>
        <y>10</y>
        <width>101</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Out_Format</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>91</width>
         <height>136</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rb_output_hex">
          <property name="text">
           <string>Hex</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_output_decimal">
          <property name="text">
           <string>Decimal</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_output_ascii">
          <property name="text">
           <string>Ascii</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_output_oct">
          <property name="text">
           <string>Oct</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_output_binary">
          <property name="text">
           <string>Binary</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTextEdit" name="OutputEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>237</y>
        <width>421</width>
        <height>171</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>200</y>
        <width>181</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Function</string>
      </property>
      <widget class="QPushButton" name="translate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Translate</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clear_input_output">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="copy_to_clipborad">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Copy to Clipboard</string>
       </property>
      </widget>
      <widget class="QPushButton" name="copy_from_clipborad">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Copy from Clipboard</string>
       </property>
      </widget>
     </widget>
     <zorder>input_groupBox</zorder>
     <zorder>Input_Label</zorder>
     <zorder>InputEdit</zorder>
     <zorder>OutputEdit</zorder>
     <zorder>output_Label</zorder>
     <zorder>output_groupBox</zorder>
     <zorder>groupBox</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
